java/ql/lib/ext/org.kohsuke.stapler.model.yml

```diff
@@ -33,9 +33,9 @@ extensions:
       - ["org.kohsuke.stapler", "StaplerRequest", True, "getSubmittedForm", "", "", "ReturnValue", "remote", "manual"]
       - ["org.kohsuke.stapler", "StaplerRequest", True, "getFileItem", "", "", "ReturnValue", "remote", "manual"]
       - ["org.kohsuke.stapler", "StaplerRequest", True, "bindParametersToList", "", "", "ReturnValue", "remote", "manual"]
-      - ["org.kohsuke.stapler", "StaplerRequest", True, "bindParameters", "", "", "Argument[0]", "remote", "manual"]
+      - ["org.kohsuke.stapler", "StaplerRequest", True, "bindParameters", "", "", "Parameter[0]", "remote", "manual"]
       - ["org.kohsuke.stapler", "StaplerRequest", True, "bindParameters", "", "", "ReturnValue", "remote", "manual"]
-      - ["org.kohsuke.stapler", "StaplerRequest", True, "bindJSON", "", "", "Argument[0]", "remote", "manual"]
+      - ["org.kohsuke.stapler", "StaplerRequest", True, "bindJSON", "", "", "Parameter[0]", "remote", "manual"]
```

Comment:
As @aschackmull said, source = `Parameter` is not a good rule of thumb. In this case, these methods bind the request parameters to some object, so it's subsequent uses of that object (i.e. it's post-update node) that we want to taint. Which means we should keep it as it was(`Argument[0]`).

Reply:
I have taken out the misguided last commit again.